README.md

```diff
@@ -13,7 +13,12 @@ For more information, see:
 
 ## Installation
 
-First install [Go 1.9.1](https://golang.org/doc/install) and [Glide](https://github.com/Masterminds/glide#install).
+First install [Go 1.9.1](https://golang.org/doc/install) and [Glide](https://github.com/Masterminds/glide#install). Note that Glide requires an explicit GOPATH to be set.
+
+```bash
+export GOPATH=$(go env GOPATH)
+mkdir -p $GOPATH
```

Comment:
why do you need the `mkdir` ?

Reply:
Makes sense to me...  I think my GOPATH was already set, so I didn't know Glide required it.

On a fresh install, though, `go` uses an "effective" `GOPATH` of `~/go`. If that directory doesn't get created automatically, it probably doesn't exist... so then our next `cd $GOPATH` line would fail.